conf/app.routes

```diff
@@ -1,3 +1,13 @@
 # microservice specific routes
 
-POST        /id         uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.id
+# TODO add endpoints for field definitions CRUD operations
+
+GET         /id/:appId/:apiContext/:apiVersion                                 @uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.id(appId: java.util.UUID, apiContext: String, apiVersion: String)
+
+GET         /application/:appId/context/:apiContext/version/:apiVersion        @uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.getSubscriptionFields(appId: java.util.UUID, apiContext: String, apiVersion: String)
+
+GET         /fieldsId/:id                                                      @uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.getSubscriptionFieldsByFieldsId(id: java.util.UUID)
```

Comment:
If we rename `id` to `fieldsId` in this route and also in the controller, it will be less confusing.
For `id` we already mean the aggregation of `applicationId`, `apiContext`, `apiVersion`

Reply:
Implemented suggested renaming. Checked code but no other occurrences.